src/shared/components/hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+
+const useLocalStorage = (key: string) => {
+  const [value, setValue] = React.useState<any>({});
+  React.useEffect(() => {
+    setValue(JSON.parse(localStorage.getItem(key) || '{}'));
```

Comment:
since localStorage may not be available if the code is run on Node.js (unless they install a package to emulate it), maybe there should be a check about whether it is present or not